app/Http/Controllers/TaskController.php

```diff
@@ -177,5 +177,22 @@ public function archived(Request $request)
    
     }
 
-   
+
+    public function getTaskByStatus($status){
```

Comment:
This method only gets all tasks, check if the tasks has a status and then get one task with intended status
This will return only one task and not all tasks.
there's no case of returning only one task with a particular status. And a sort method already exists [Check Here](https://github.com/zurichat/zc_plugin_todo/blob/develop/app/Http/Controllers/TaskController.php)
check the figma design of cases of return task based on status [ Here](https://www.figma.com/file/IDf0uzudB91FmWaCMcVaYv/SOFTWARE-ARCHITECTURE?node-id=199%3A826)